### DISCUSSION TITLE
How to set the operation mode of the reservoirs by ourselves

### DISCUSSION THREAD

**Discussion by walter-git** [iiasa/CWatM#47] [Q&A] [answered]
Hi everyone,
Hope you all are doing well.
I want to set the operation mode of the reservoirs based on the observations.
But I don't know how to operate in CWATM.
Please help me this regard.
Thank you very much.
Meishui Li
Beijing Normal University

**Comment by mikhailsmilovic** [comment, marked as answer]
Guten Tag, great question! This determines **reservoir releases as a fraction of live storage**. These are input as 366-day arrays for each reservoir, each reservoir input as a column in an Excel sheet. There are two types of releases: _Reservoirs_downstream_, which discharge downstream into the river, and _Reservoirs_supply_, which is the maximum amount used to satisfy command area demands. In the following discussion, we focus only on _Reservoirs_downstream_.

Each reservoir with tailored management is included with its ID in the header of a column, followed by 366 live storage fractions. 

Include all the reservoirs you wish to tailor the operations for. The reservoirs that are not included in the Excel will be managed in the standard CWatM way, with different levels of release determined with" three storage limits (conservative, normal, flood) and three outflow functions (minimum, normal, non-damaging)."

### Procedure
In [OPTIONS]: 
reservoir_releases_in_Excel_settings = True

Then, in the [FILE_PATHS] section,
Excel_settings_file = _path to Excel settings file_
We give an example here: [cwatm_settings_reservoirs.xlsx](https://github.com/iiasa/CWatM/files/13283268/cwatm_settings_reservoirs.xlsx)

**Reply by walter-git** [reply]
Thank you very much！

**Comment by mikhailsmilovic** [comment]
Great this helps. Let us know how it goes!
